{
  try {
    tester.startPage(AbortExceptionPage.class,new PageParameters().set("trigger",true));
    Assert.assertEquals((Integer)1234,tester.getLastResponse().getStatus());
  }
 catch (  RuntimeException x) {
    final Throwable reason=x.getCause();
    Assert.assertEquals(reason.getClass(),AbortWithHttpErrorCodeException.class);
    Assert.fail("this must not happen (we expect a redirect happen here and handled by wicket request processor)");
  }
}
